tools/run_tests/run_interop_tests.py

```diff
@@ -777,10 +777,11 @@ def cloud_to_prod_jobspec(language,
     ]
     if transport_security == 'tls':
         transport_security_options = ['--use_tls=true']
-    elif transport_security == 'google_default_credentials' and language == 'c++':
+    elif transport_security == 'google_default_credentials' and (language == 'c++' or language == 'go'):
```

Comment:
Actually, same `str(language)` change also needs to be made here.

Also while we're at it, cosmetic nit:

Here and everywhere else, we should probably use:

```
str(language) in ['c++', 'go']
```

instead of the current 

```
`language == 'c++' or language == 'go'`
```